### PR TITLE
readme: README.md, spelling, grammar

### PR DESCRIPTION
**Pull request by SergeCroise**
- all projec**ts**
- two releas**es** ...
- develo**ping**

see issue #270

### CODE REVIEW

**Review by clarani — APPROVED** [review]

**Comment by SharglutDev** [comment]
maybe you can try a rebase + push to see if it reruns the CI ?

**Comment by emersion** [comment]
Seems like CI doesn't kick in for external contributors? An attempt to fix this: https://github.com/OpenRailAssociation/osrd-ui/pull/432

**Comment by emersion** [comment]
Can you try rebasing on top of the latest commit from `dev`?

**Comment by emersion** [comment]
We don't use merge commits. Can you rebase instead?

**Comment by SergeCroise** [comment]
hi @emersion ,
is my force-push ok?
or should I create a new PR?
thanks for your help!

**Comment by emersion** [comment]
Yes, perfect, thanks!

**Comment by emersion** [comment]
Ah, seems like CI isn't happy about the commit title. You can probably leave out "README.md" since the prefix is already "readme: ".

**Comment by SergeCroise** [comment]
> Ah, seems like CI isn't happy about the commit title. You can probably leave out "README.md" since the prefix is already "readme: ".

you are right!

**Comment by emersion** [comment]
Ah, seems like you've added a new commit instead of amending the existing one.

**Comment by SergeCroise** [comment]
> Ah, seems like you've added a new commit instead of amending the existing one.
wanted to squash both commits to one with adequate title

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ We do not change the version on local package versions, we keep the file to the
 which is not a real version and can be easily identified as a development version.
 
 The version numbers for our releases are solely managed through git tags. It implies that when we
-update a single package, we release all the project together : even if there are no changes between
-two release (let's say we update ui-icons but nothing else, we would make a release that would
+update a single package, we release all the projects together : even if there are no changes between
```
Comment (SergeCroise) [inline README.md]:
```suggestion
update a single package, we release all the projects together: even if there are no changes between
```

**Comment by SergeCroise** [comment]
@emersion  , if you agree, I close this request and create a new one.

**Comment by emersion** [comment]
Ah… I'd suggest looking at https://git-rebase.io to learn how to use `git rebase`. It shouldn't be necessary to close/reopen new PRs.

**Comment by emersion** [comment]
More specifically, to fix this branch, you can `git rebase -i dev`, then change the first "pick" to "reword" (to edit the commit message) and the rest to "squash" (to squash all but the first commit into the first commit), then force-push and re-open this PR.

**Comment by SergeCroise** [comment]
@emersion , please review!

**Comment by emersion** [comment]
Thanks for bearing with us :)

**Pull request merged**